requirements-docs.txt

```diff
@@ -4,7 +4,7 @@ jinja2<3.1
 
 
 docutils>=0.10,<0.17
-Sphinx==1.3.2
+Sphinx==1.4.9
```

Comment:
What is this bump for?

Reply:
Setting the document language was introduced in 1.4.0
https://www.sphinx-doc.org/en/master/changes.html#:~:text=Add%20%E2%80%9Clanguage%E2%80%9D%20attribute%20to%20html%20tag%20in%20the%20%E2%80%9Cbasic%E2%80%9D%20theme

Reply:
Got it. That's what I assumed it as based on the commit, but I didn't see it in the changelog.